woocommerce/woocommerce-android
# Feature/7510 account creation network

Closes: #7510 


Do not merge until [FluxC changes](https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2544) have been merged

### Description
Adds the needed logic to create a new account using `https://public-api.wordpress.com/rest/v1.1/users/new?` endpoint. 

Error handling will be handled in an upcoming PR

### Testing instructions
Follow the demo steps to create a new account: 

- Click on "Get Started" 
- Input new account credentials 
- Click on "Get started" again
- If account is created successfully you will land in the next step of the flow, which is an empty fragment right now

https://user-images.githubusercontent.com/2663464/196914123-bff7c001-10a7-4e7a-99ea-008421872e2a.mp4

- Proceed killing the app and reopening it again to check that you are successfully logged in. 

https://user-images.githubusercontent.com/2663464/196914243-454aac24-b4cd-46a8-8707-97c1c573838e.mp4
